src/js/components/widget.jsx

```diff
@@ -49,6 +53,11 @@ export class WidgetComponent extends Component {
 
     onClick = () => {
         resetUnreadCount();
+        this.setState({showAnimation: true});
```

Comment:
I would do this in `openWidget` and `closeWidget` function in the app service instead. This would ensure it's done whenever the action is done.